include/aie/Dialect/AIEVec/Pipelines/Passes.h

```diff
@@ -0,0 +1,70 @@
+//===- Passes.h - AIE Vector pipeline entry points --------------*- C++ -*-===//
```

Comment:
Why do you comment line 17-68?

Reply:
We don't have a pipeline yet, this was here as a reference for me. I meant to remove all of these, this one slipped through. Good catch, thanks!